tests/std/tests/P0355R7_calendars_and_time_zones_formatting/test.cpp

```diff
@@ -884,6 +888,13 @@ void test_zoned_time_formatter() {
     assert(format(STR("{:%g %G %U %V %W}"), zt) == STR("21 2021 16 16 16"));
 }
 
+template <typename CharT>
+void test_localized() {
+    auto loc = locale("ru-RU");

```

Comment:
We conventionally avoid the "Almost Always Auto" pattern of constructing a temporary and storing it with `auto`. Directly naming the variable's type is simpler:
```suggestion
    locale loc("ru-RU");
```